common/helpers.js

```diff
@@ -257,32 +316,34 @@ function getFileInfoFromDrsResponse(drsResponse, googleServiceAccount) {
     const updatedTimeIso = updatedTime ? new Date(updatedTime).toISOString() : null;
     const gsUrl = getGsUrlFromDrsObject(drsResponse);
     const [bucket, name] = parseGsUri(gsUrl);
-    const md5Checksum = getMd5Checksum(checksums);
+    const hashesMap = getHashesMap(checksums);
     const signedUrl = null; // Not included currently when returning only the drs metadata
 
-    return convertToFileInfoResponse(
+    return new MarthaV3Response(
         mimeType,
         size,
         createdTimeIso,
         updatedTimeIso,
-        md5Checksum,
         bucket,
         name,
         gsUrl,
         googleServiceAccount,
         signedUrl,
```

Comment:
If we're forking the response types, what do you @salonishah11 and other reviewers think of ditching the `signedUrl` for now?

We can always add it back if we re-unify the response types, or actually start returning a signed URL in a future ticket.

My dream of a single response type was to ease transitions for clients, but as the servers are each doing their own thing... Marth'll do what what she has to do.

Reply:
I agree. Since the responses are already, there is no harm in removing `signedUrl` from `/martha_v3` response for now as it is always an empty string.